WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadStarter.kt

```diff
@@ -165,6 +167,7 @@ class UploadStarter @Inject constructor(
                     .toList()
                     .forEach { post ->
                         AppLog.d(AppLog.T.POSTS, "UploadStarter for post title: ${post.title}")
+                        dispatcher.dispatch(UploadActionBuilder.newIncrementNumberOfAutoUploadAttemptsAction(post))
```

Comment:
We want to increment the number of auto upload attempts only at this place - when we actually initiate the auto upload.

Reply:
⚠️ I don't see this being covered in `UploadStarterTest`. Did I miss it?